src/store/getters/state-getters.ts

```diff
@@ -64,7 +64,12 @@ export const getFilingName = (state: StateIF): FilingNames => {
   switch (getFilingType(state)) {
     case FilingTypes.INCORPORATION_APPLICATION: return FilingNames.INCORPORATION_APPLICATION
     case FilingTypes.REGISTRATION: return FilingNames.REGISTRATION
-    case FilingTypes.VOLUNTARY_DISSOLUTION: return FilingNames.VOLUNTARY_DISSOLUTION
+    case FilingTypes.VOLUNTARY_DISSOLUTION:
+      if (getEntityType(state) === CorpTypeCd.SOLE_PROP) {
```

Comment:
You should use `isTypeSoleProp(state)` instead.

Reply:
Thank you , I never know we had a getter for it 👍